### PR TITLE
[tr] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by ayyucekizrak**
I have started translate to Turkish. Still in progress.

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you for your work @ayyucekizrak! 

@yavuzKomecoglu would you like to be the reviewer for this translation?

**Comment by ayyucekizrak** [comment]
Its my pleasure!

Yavuz Komecoglu is fine for me, lets do it !

Ayyuce's iPhone sent

Shervine Amidi <notifications@github.com<mailto:notifications@github.com>> şunları yazdı (24 Oca 2019 21:54):


Thank you for your work @ayyucekizrak<https://github.com/ayyucekizrak>!

@yavuzKomecoglu<https://github.com/yavuzKomecoglu> would you like to be the reviewer for this translation?

File: tr/refresher-probability.md [hunk 1]
```diff
@@ -1,204 +1,204 @@
...
 
-&#10230;
+&#10230; 32. Olasılık Dağılımları
 
 <br>
 
 **33. Chebyshev's inequality ― Let X be a random variable with expected value μ. For k,σ>0, we have the following inequality:**
 
-&#10230;
+&#10230; 33. Chebyshev'in eşitsizliği - X'in beklenen değeri value olan rastgele bir değişken olmasına izin verin. K, σ>0 için aşağıdaki eşitsizliği elde edilir:
```
Comment (basakbuluz) [inline tr/refresher-probability.md]: 33. Chebyshev'in eşitsizliği - X beklenen değeri μ olan rastgele bir değişken olsun. K, σ>0 için aşağıdaki eşitsizliği elde edilir:

**Comment by shervinea** [comment]
Thank you both, again! Merging this PR and adding you to the contributors list.

**Pull request merged**